.github/workflows/scripts/release-changelog.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Helper script for inserting NEXT_CHANGELOG contents into changelog and readme files.
+ * Helper script for inserting NEXT_CHANGELOG contents into changelog.
```

Comment:
I think we want to say `Helper script for inserting NEXT_CHANGELOG contents into readme.txt file.`

Reply:
Yep, had some comments backward, fixed those, thanks.